public/apps/account/account-app.tsx

```diff
@@ -38,7 +38,7 @@ function tenantSpecifiedInUrl() {
 
 export async function setupTopNavButton(coreStart: CoreStart, config: ClientConfigType) {
   const authType = config.auth?.type;
-  let currAuthType = '';
```

Comment:
Is empty string the equivalent of AuthType.BASIC? I think this should remain as is.